tutorials/get-started-with-dqlite-in-go.md

```diff
@@ -0,0 +1,23 @@
+This step-by-step guide walks you through the setup of an Dqlite example application. 
+By the end of this tutorial you will have a fully-functional Dqlite cluster deployed on your machine and you'll be able to interact with it through a HTTP client.
+
+## What you’ll need:
+
+* A workstation, e.g. a laptop
+* Some familiarity with Go
+
+## What you’ll do:
+
+* [Setup your development environment](tbd)
+* [Create an example application](tbd)
+
+## Next steps:
+
+In the next step we'll [setup your development environment](tbd).
+But there is a lot more to explore:
+
+| IF YOU ARE WONDERING... | VISIT...                                                            |
+|-------------------------|---------------------------------------------------------------------|
+| "How do I...?           | [How-to docs](tbd)             |
+| "What is ...?           | [Reference docs](tbd)    |
+| "Why...?", "So what?"   | [Explanation docs](tbd) |
```

Comment:
This is a good idea, but unfortunately it won't work with the current setup of the docs. There aren't any pages to link to here, since How-to/Reference/Explanation are just headings for now and not linked to a page.
So I would leave out the table for now.